remappings.txt

```diff
@@ -0,0 +1,8 @@
+solady/=lib/solady/
+solemate/=/lib/solemate/src/
+@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/
+@openzeppelin/contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/contracts/
+forge-std/=lib/forge-std/src/
+erc721a/contracts=lib/ERC721a/contracts/
```

Comment:
This will work on mac, on linux the lowercase "a" in `lib/ERC721a/contracts` will not map properly because of case sensitivity. 

Probably not a problem in local development on a mac but in a CICD env running an ext4 filesystem this will throw a file not found.

Suggested: `lib/ERC721a/contracts`